types/deno/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "private": true,
+    "name": "@types/deno",
+    "version": "2.0.9999-rc.1",
```

Comment:
This shouldn't be a legal version number; I have no idea why the tooling is accepting it. We don't allow pre release versions on DT.

Reply:
Oh, how do people usually release types for pre-release versions? I guess I could just make this `2.0.9999` in this case because we're just creating the package.

Reply:
They generally don't; we don't allow types for prerelease versions.

The only exceptions are cases that were mistakenly let through and so have versions which mismatch those of upstream.

Reply:
(This comes down to the tooling not actually doing the right thing for prereleases, plus that ATA consumes DT so not supporting prereleases in the publishing means we'd download the wrong `@latest`, etc; uncharted territory a bit)

Reply:
Updated to remove the pre-release.